### PR TITLE
Update order with unit test

### PR DESCRIPTION
**Pull request by ankitdas13**
for edit : https://github.com/razorpay/razorpay-node/blob/master/documents/order.md#update-order
Link : https://razorpay.com/docs/api/orders/#update-order

for fetchorderTransfer :  https://github.com/razorpay/razorpay-node/blob/master/documents/transfer.md#fetch-transfer-for-a-payment
Link : https://razorpay.com/docs/api/payments/route/#fetch-transfer-for-an-order

### CODE REVIEW

File: lib/razorpay/order.rb [hunk 1]
```diff
@@ -25,5 +25,13 @@ def payments(options = {})
       r = self.class.request
       r.request :get, "/orders/#{id}/payments", options
     end
+    
+    def self.edit(id, options = {})
+      request.patch id, options
+    end
+
+    def self.fetchTransferOrder(id)
```
Comment (arjunmenon) [inline lib/razorpay/order.rb:33]: nit: snakecase

File: test/razorpay/test_order.rb [hunk 2]
```diff
@@ -44,5 +44,27 @@ def test_order_payments_should_be_fetched
...
+      assert_instance_of Razorpay::Order, order, 'order not an instance of Razorpay::Order class'
+      assert_equal @order_id, order.id, 'order IDs do not match'
+   end
+
+   def test_fetch_order_transfers
+    stub_get("#{BASE_URI}orders/#{@order_id}/?expand[]=transfers&status", 'fake_order_transfers')
+    order = Razorpay::Order.fetchTransferOrder(@order_id)
+    assert_instance_of Razorpay::Order, order, 'order not an instance of Razorpay::Order class'
+    assert_equal @order_id, order.id, 'order IDs do not match'
+    refute_empty order.transfers
```
Comment (arjunmenon) [inline test/razorpay/test_order.rb:67]: check `order.transfers.items`. `transfers` has other attr which is not indicative of actual item collection

File: lib/razorpay/order.rb (hunk 1 above)
Comment (arjunmenon) [reply lib/razorpay/order.rb:33]: Missing doc for this

Comment (ankitdas13) [reply lib/razorpay/order.rb:33]: i have doc and change function in snakecase

File: test/razorpay/test_order.rb (hunk 2 above)
Comment (ankitdas13) [reply test/razorpay/test_order.rb]: i have checked order.transfers.items and added assertion for transfers.items

File: lib/razorpay/order.rb (hunk 1 above)
Comment (arjunmenon) [reply lib/razorpay/order.rb]: SDK doc is missing. What you added is a link to the doc. - https://github.com/razorpay/razorpay-ruby/pull/132/commits/33873b01c43c4a8baf9ab46a6836a4f9ed6d7015#diff-4e51636c699f2c237011920308f14d500ee93b09bef25443e5c2720171900954R35

Comment (arjunmenon) [reply lib/razorpay/order.rb]: \<internal\> https://razorpay.slack.com/archives/C029ZL11PNK/p1649335900167959?thread_ts=1647927334.987909&cid=C029ZL11PNK

**Review by arjunmenon — APPROVED** [review]
LGTM

**Pull request merged**